src/components/organisms/NavBarSchedule/NavBarSchedule.scss

```diff
@@ -126,3 +130,28 @@ $schedule-btn--height: 51px;
     padding: $spacing--sm 0px;
   }
 }
+
+@media screen and (max-width: $media-breakpoint--md) {
+  .NavBarSchedule {
+    width: 100%;
+  }
+
+  .NavBarWrapper {
+    &__download-buttons {
+      width: 100%;
+    }
+
+    &__close-button {
+      height: 45px;
+      width: 45px;
```

Comment:
`$constants`?

Reply:
fixed in               onClick={downloadPersonalEventsCalendar}
              customClass="NavBarWrapper__download-schedule-btn"